hack/update-generated-proto-bindings-dockerized.sh

```diff
@@ -14,16 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This script generates `*/api.pb.go` from the protobuf file `*/api.proto`.
-# Example:
-#   kube::protoc::generate_proto "${DYNAMIC_RESOURCE_ALLOCATION_ALPHA}"
+# This script generates `*/api.pb.go` files from protobuf files `*/api.proto`.
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
 KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd -P)"
-DYNAMIC_RESOURCE_ALLOCATION_V1ALPHA1="${KUBE_ROOT}/staging/src/k8s.io/kubelet/pkg/apis/dra/v1alpha1/"
 
 source "${KUBE_ROOT}/hack/lib/protoc.sh"
-kube::protoc::generate_proto "${DYNAMIC_RESOURCE_ALLOCATION_V1ALPHA1}"
+
+for api; do
+    # This can't use `git ls-files` because it runs in a container without the
+    # .git dir synced.
+    find "${api}" -type f -name "api.proto" \
```

Comment:
should we fail if this finds zero results, rather than silently doing nothing?

Reply:
Good question - we haven't thus far.  New commit coming.